confignode/src/main/java/org/apache/iotdb/confignode/manager/load/LoadManager.java

```diff
@@ -281,6 +286,19 @@ public void removeNodeHeartbeatHandCache(Integer nodeId) {
     heartbeatCacheMap.remove(nodeId);
   }
 
+  public List<TConfigNodeLocation> getOnlineConfigNodes() {
+    getNodeManager()
+        .getRegisteredDataNodes()
+        .forEach(
+            registerConfigNode -> {
+              if (heartbeatCacheMap.get(registerConfigNode.getConfigNodeId()).getNodeStatus()
+                  == NodeStatus.Unknown) {
+                onlineConfigNodes.remove(registerConfigNode);
+              }
+            });
+    return onlineConfigNodes;
```

Comment:
New List<TConfigNodeLocation> to store the result, thus we don't have to maintain the onlineConfigNodes field in LoadManager